lib/assembly-image/version.rb

```diff
@@ -1,10 +0,0 @@
-# frozen_string_literal: true
-
-# Main Assembly namespace
-module Assembly
-  # Main Image class
-  class Image
-    # Gem version
-    VERSION = '1.8.0'
-  end
-end
```

Comment:
because `Assembly::Image` is now a subclass of a class in the `assembly-objectfile` gem, we can't have the VERSION file in the appropriate Assembly::Image namespace AND have it be part of the gemspec.